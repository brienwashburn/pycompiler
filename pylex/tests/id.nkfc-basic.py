# This identifier should be normalized in the tokenized output, assumes your pylex outputs UTF-8 (Racket default)
ず = 3
