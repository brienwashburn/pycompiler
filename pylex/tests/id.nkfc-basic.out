(ID "ず")
(PUNCT "=")
(LIT 3)
(NEWLINE)
(ENDMARKER)
